### DISCUSSION TITLE
Free SQL IDE for Kyuubi

### DISCUSSION THREAD

**Discussion by ryanhamilton** [apache/kyuubi#6460] [Polls]
Hi,

I'm the author of qStudio a Free SQL IDE that bundles a Kyuubi driver (screenshot below).
If anyone would be willing to send a quick guide to running Kyubi and querying it from qStudio I would love to add the guide to our website. You can see a similar guide for QuestDB here:
https://www.timestored.com/qstudio/database/questdb

Or if you try it and find problems, post on our github issues and I will help fix it:
https://github.com/timeseries/qstudio/issues

![image](https://github.com/apache/kyuubi/assets/7273208/fd25054c-5dc8-435e-960b-50294919aa41)

https://github.com/apache/kyuubi/blob/master/docs/quick_start/quick_start_with_jdbc.md

**Comment by pan3793** [comment]
Good to know there is an awesome SQL GUI tool, thanks for sharing!

**Comment by pan3793** [comment]
If you want, you can also add a docs on the Kyuubi website to guide users to use qStudio to integrate with Apache Kyuubi, just like we did for DBeaver https://kyuubi.readthedocs.io/en/master/client/bi_tools/dbeaver.html

**Comment by ryanhamilton** [comment]
Is there a PR that serves as a good example?

**Reply by pan3793** [reply]
the corresponding source code of docs is located at
https://github.com/apache/kyuubi/blob/master/docs/client/bi_tools/dbeaver.rst, and you are free to choose either rst or markdown format to write new docs

**Reply by pan3793** [reply]
one tip: if you upload img, please try to compress it as much as possible, otherwise the size of the codebase repo may grow fast.
